### PR TITLE
Changes in Self Assessments Docs..

### PR DESCRIPTION
**Pull request by rakeshateka**
SCR-20240920-rifl

### CODE REVIEW

File: api-reference/general-tools/assessment/component-info/component.yml [hunk 1]
```diff
@@ -79,99 +34,364 @@ paths:
...
               schema:
-                type: string
-                example: same-origin
+                type: object
+              example:
+                success: true
+                data:
+                  autosuggest:
+                    question:
+                      "$schema": http://json-schema.org/draft-07/schema#
```
Comment (mayank-ekacare) [inline api-reference/general-tools/assessment/component-info/component.yml:46]:
```suggestion
                      schema:
```

this should work right?

File: api-reference/general-tools/assessment/initialization/initialize-assessment.yml [hunk 2]
```diff
@@ -38,132 +41,50 @@ paths:
...
-                example: same-origin
           content:
             application/json:
               schema:
                 type: object
               example:
                 data:
                   init_hash: sn_1212121212
-        '400':
+        '5XX':
```
Comment (mayank-ekacare) [inline api-reference/general-tools/assessment/initialization/initialize-assessment.yml:80]:
```suggestion
        '4XX':
```

5XX says internal server error's please have a look make changes accordingly

Comment (rakeshateka) [reply api-reference/general-tools/assessment/initialization/initialize-assessment.yml]: We have not finalized the error response yet. Will update it later once the error dictinary format and error codes gets finalized.
This one is written just to give an idea to frontend team (Gautham).

File: api-reference/general-tools/assessment/component-info/component.yml (hunk 1 above)
Comment (rakeshateka) [reply api-reference/general-tools/assessment/component-info/component.yml:46]: It's fine.

**Review by mayank-ekacare — APPROVED** [review]

**Pull request merged**